src/cocoa/toga_cocoa/widgets/table.py

```diff
@@ -163,3 +162,32 @@ def scroll_to_row(self, row):
     def rehint(self):
         self.interface.intrinsic.width = at_least(self.interface.MIN_WIDTH)
         self.interface.intrinsic.height = at_least(self.interface.MIN_HEIGHT)
+
+    def _add_column(self, heading, accessor):
+        column_identifier = at(accessor)
+        self.column_identifiers[id(column_identifier)] = accessor
+        column = NSTableColumn.alloc().initWithIdentifier(column_identifier)
+        self.table.addTableColumn(column)
+        self.columns.append(column)
+
+        cell = TogaIconCell.alloc().init()
+        column.dataCell = cell
+
+        column.headerCell.stringValue = heading
+
+    def add_column(self, heading, accessor):
+        self._add_column(heading, accessor)
+        self.table.sizeToFit()
+
+    def remove_column(self, accessor):
+        column = self.table.tableColumnWithIdentifier(accessor)
```

Comment:
This seems inconsistent - the object is created with an Objective C reference to a column identifier, but removed with the accessor (a Python string)?

Reply:
I fixed!